public/test/setupTests.ts

```diff
@@ -20,3 +21,6 @@ i18next.use(initReactI18next).init({
   returnEmptyString: false,
   lng: 'en-US', // this should be the locale of the phrases in our source JSX
 });
+
+// Increase testing-library "waitFor" methods timeout from 1000ms
+configure({ asyncUtilTimeout: 2000 });
```

Comment:
Wouldn't it be enough to increase timeout only in those tests that are affected?

Reply:
Nit: maybe it worth sharing in the comment which tests required us to increase the limit (if there are not too many of them)?

Reply:
I'm not sure we can do it for individual tests. I went with the global because in CI jest tests would randomly fail with cannot find element etc. I'm guessing it's because we no longer mock monaco and kusto in jest and these libs make things a little heavier during tests. As a timeout I don't think this has a major impact on time across all tests. Any `waitFor` that would previously resolve within the 1000ms would continue to within that time frame. If anything it might make our jest tests more robust in CI.

I can add a comment along the lines of `our tests are heavy due to monaco and kusto so we need to increase the default timeout`?

Reply:
I see, yeah, I think that comment makes sense. (I thought that there are certain test-case that we identified failing).

Reply:
Have updated comment with:

```
// our tests are heavy in CI due to parallelisation and monaco and kusto
// so we increase the default timeout to 2secs to avoid flakiness
```